Procfile

```diff
@@ -0,0 +1 @@
+web: python manage.py runserver 0.0.0.0:$PORT
```

Comment:
Yeah, that did the trick. I also had to use the heroku build packs for node.js and python